### ISSUE TITLE
Error loading syntax file

### ISSUE THREAD

**Issue by KES777** [Blaizer/ModernPerl-sublime#36]
![image](https://user-images.githubusercontent.com/799021/138137160-565f21c0-bf1f-4855-84db-1c18d454d464.png)

**Comment by nugged** [comment]
I now have the same in Sublime Merge after some updates came to both tools...

![image](https://user-images.githubusercontent.com/295324/138153053-af6ebc80-01c0-4e6f-961b-daaa2378ebbf.png)

**Comment by sm-kc** [comment]
Same here. Sublime Text build 4121 on MacOS; system Perl is 5.30.

**Comment by brabes** [comment]
Related [SublimeText forum post](https://forum.sublimetext.com/t/error-loading-syntax-file-c-improved-package/61696).

Seems like it's a bug in ST4 with `.tmLanguage` syntax files, and the solution is to convert to using `.sublime-syntax` format, but it appears that's far from trivial.

**Comment by nugged** [comment]
Yes, it started to work after the last update for ST4 (build 4124),
I removed the ModernPerl package and installed it again, and... it started to work!

![image](https://user-images.githubusercontent.com/295324/145647045-609fec75-a1c2-4054-a412-34ed27003b03.png)

I did the same operation on a small own Mac and on iMac... now it works on all.

@KES777, @sm-kc, can you confirm too? (🤞)

**Comment by KES777** [comment]
@nugged Hi, Andrew.
I reinstall `Modern perl` package. Error is still here, but my ST version is 4122 yet.

I upgrade sublime to 4124, reinstall plugin again and error gone

**Comment by sm-kc** [comment]
Hello, happy new year! I'm pleased to confirm that ModernPerl is working perfectly on Sublime Text build 4126 on MacOS.
